packages/runtime-tools-enveloped-components/package.json

```diff
@@ -35,15 +36,20 @@
     "@kie-tools/runtime-tools-components": "workspace:*",
     "@kie-tools/runtime-tools-gateway-api": "workspace:*",
     "@kie-tools/serverless-workflow-combined-editor": "workspace:*",
+    "@patternfly/react-code-editor": "^4.82.113",
```

Comment:
In kie-tools, we have our own monaco-editor, so the react-code-editor dependency should not be added. Instead, a similar approach to the cloud-event-form enveloped component should be used, replacing the react-code-editor with the appropriate editor, which is inside the RequestDataEditor in runtime-tools-component.